src/js/view/controls/play-display-icon.js

```diff
@@ -43,25 +43,33 @@ export default class PlayDisplayIcon {
                 iconDisplay.removeAttribute('aria-label');
             }
 
-            this.toggleIdleClass(oldState, newState, idleButton);
+            this.toggleIdleClass(oldState, newState, idleButtonText);
         });
 
-        this.toggleIdleClass('', 'idle', idleButton);
+        this.toggleIdleClass('', 'idle', idleButtonText);
     }
 
     element() {
         return this.el;
     }
 
-    toggleIdleClass(oldState, newState, idleButton) {
-        if (!(idleButton === 'stroke' || idleButton === 'fill' || idleButton === 'label')) {
+    toggleIdleClass(oldState, newState, idleButtonText) {
+        if (idleButtonText !== 'Click to Play' && idleButtonText !== 'Play' && idleButtonText !== 'Watch Now') {
```

Comment:
This would be a good use case for some regex:
`!/^(click to play|play|watch now)$/i.test(idleButtonText)`